addons/account/views/account_payment_view.xml

```diff
@@ -175,6 +175,11 @@
                         The selected payment method requires a bank account but none is set on
                         <button class="oe_link alert-link" type="object" name="action_open_destination_journal" style="padding: 0; vertical-align: baseline;">the destination journal</button>.
                     </div>
+                     <div class="alert alert-warning mb-0 w-100 d-flex align-items-center gap-1" invisible="not has_duplicated_payments or state!='draft'" role="alert">
+                        <field name="duplicated_payment_ids" widget="x2many_buttons" string="Duplicated Payments"/>
+                         <span invisible="has_duplicated_payments == 1">with the same partner, amount and date already exist.</span>
```

Comment:
Like i say by using the x2many_buttons you can override the template to add this without having to add a field i think :smile: